packages/compass/src/main/get-os-info.ts

```diff
@@ -0,0 +1,56 @@
+import os from 'os';
+import { promises as fs } from 'fs';
+
+type OsInfo = {
+  os_type: string;
+  os_version: string;
+  os_arch: string;
+  os_release: string;
+  os_linux_dist?: string;
+  os_linux_release?: string;
+};
+
+async function getLinuxInfo(): Promise<{
+  os_linux_dist: string;
+  os_linux_release: string;
+}> {
+  try {
+    const releaseFile = '/etc/os-release';
+    const etcRelease = await fs.readFile(releaseFile, 'utf-8');
+
+    const releaseKv = etcRelease
+      .split('\n')
+      .map((l) => l.trim())
+      .filter(Boolean)
+      .map((l) => l.split('='));
```

Comment:
Super tiny style nit: if you do `Object.fromEntries`  or `new Map` here, you'd get an easier access to these values in the code below

```suggestion
    const releaseKv = Object.fromEntries(
      etcRelease
        .split('\n')
        .map((l) => l.trim())
        .filter(Boolean)
        .map((l) => l.split('='))
    );
```

Reply:
That's a nice trick! didn't even know  `Object.fromEntries` was a thing